### ISSUE TITLE
Not find "tools/mix_data_test_dancetrack.py".

### ISSUE THREAD

**Issue by wjy327** [RainBowLuoCS/DiffusionTrack#9]
Dear author thank you very much for your excellent work! I  can not find "tools/mix_data_test_dancetrack.py".Can you help me?Thank you!

**Comment by RainBowLuoCS** [comment]
We're sorry, we don't have this file. We've modified the Readme file. You can use the Dancetrack training set to train and evaluate on the test set, thank you for your reminder

**Comment by wjy327** [comment]
Thank you for your help.
